### PR TITLE
refactor(core): fix formatting and address comments

### PR DESCRIPTION
**Pull request by kara**
Should address @IgorMinar's comments and @robwormald's comment. Formatted with current clang-format and ran buildifier. (Tried to run new clang-format, but it appears to not honor `clang-format off`?)

cc @jelbourn

### CODE REVIEW

File: modules/benchmarks/e2e_test/largetable_perf.ts [hunk 1]
```diff
@@ -65,10 +65,10 @@ describe('largetable benchmark perf', () => {
         }).then(done, done.fail);
       });
 
-      it('should run for iv', (done) => {
+      it('should run for ivy', (done) => {
         runTableBenchmark({
-          id: `largeTable.iv.${worker.id}`,
-          url: 'all/benchmarks/src/largetable/iv/index.html',
+          id: `largeTable.ivy.${worker.id}`,
+          url: 'all/benchmarks/src/largetable/ivy/index.html',
```
Comment (mhevery) [inline modules/benchmarks/e2e_test/largetable_perf.ts]: The path's should say `render3` not `ivy`. I would also not include `ivy` anywhere in the source as it is a code name. Just say render3.

**Review by mhevery — CHANGES_REQUESTED** [review]

File: modules/benchmarks/e2e_test/largetable_perf.ts (hunk 1 above)
Comment (marclaval) [reply modules/benchmarks/e2e_test/largetable_perf.ts]: This benchmark is actually `iv`, the original prototype from Bertrand.

Comment (mhevery) [reply modules/benchmarks/e2e_test/largetable_perf.ts]: good, point, we should keep it as `iv` than

File: packages/core/src/render3/instructions.ts [hunk 2]
```diff
@@ -82,7 +82,7 @@ let ngStaticData: NgStaticData;
 /**
  * State of the current view being processed.
  */
-let currentView: ViewState = createViewState(null !, null !, []);
+var currentView: ViewState = createViewState(null !, null !, []);
```
Comment (mhevery) [inline packages/core/src/render3/instructions.ts]: why change `let` to `var`?

**Review by mhevery — CHANGES_REQUESTED** [review]
see comments

**Comment by kara** [comment]
@mhevery Should be ready

**Comment by kara** [comment]
Closing, will make changes against master

**Pull request closed without merging**